browser/ui/brave_shields_data_controller.cc

```diff
@@ -49,7 +52,11 @@ int BraveShieldsDataController::GetTotalBlockedCount() {
 bool BraveShieldsDataController::GetIsBraveShieldsEnabled() {
   auto* map = HostContentSettingsMapFactory::GetForProfile(
       web_contents()->GetBrowserContext());
-  return brave_shields::GetBraveShieldsEnabled(map, web_contents()->GetURL());
+  return brave_shields::GetBraveShieldsEnabled(map, GetCurrentSiteURL());
+}
+
+GURL BraveShieldsDataController::GetCurrentSiteURL() {
+  return web_contents()->GetURL();
```

Comment:
As @bridiver noted - https://bravesoftware.slack.com/archives/C7VLGSR55/p1642016039043400,
`GetURL()` is deprecated method.`GetLastCommittedURL()` is more appropriated one.